packages/expo-updates/android/src/androidTest/java/expo/modules/updates/manifest/ManifestFactoryTest.java

```diff
@@ -13,72 +13,75 @@
 
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
 @RunWith(AndroidJUnit4ClassRunner.class)
 public class ManifestFactoryTest {
 
   private String legacyManifestJson = "{\"sdkVersion\":\"39.0.0\",\"id\":\"@esamelson/native-component-list\",\"releaseId\":\"0eef8214-4833-4089-9dff-b4138a14f196\",\"commitTime\":\"2020-11-11T00:17:54.797Z\",\"bundleUrl\":\"https://d1wp6m56sqw74a.cloudfront.net/%40esamelson%2Fnative-component-list%2F39.0.0%2F01c86fd863cfee878068eebd40f165df-39.0.0-ios.js\"}";
-  private String newManifestJson = "{\"manifest\":{\"runtimeVersion\":\"1\",\"id\":\"0eef8214-4833-4089-9dff-b4138a14f196\",\"createdAt\":\"2020-11-11T00:17:54.797Z\",\"launchAsset\":{\"url\":\"https://d1wp6m56sqw74a.cloudfront.net/%40esamelson%2Fnative-component-list%2F39.0.0%2F01c86fd863cfee878068eebd40f165df-39.0.0-ios.js\",\"contentType\":\"application/javascript\"}}}";
+  private String newManifestJson = "{\"runtimeVersion\":\"1\",\"id\":\"0eef8214-4833-4089-9dff-b4138a14f196\",\"createdAt\":\"2020-11-11T00:17:54.797Z\",\"launchAsset\":{\"url\":\"https://d1wp6m56sqw74a.cloudfront.net/%40esamelson%2Fnative-component-list%2F39.0.0%2F01c86fd863cfee878068eebd40f165df-39.0.0-ios.js\",\"contentType\":\"application/javascript\"}}";
   private String bareManifestJson = "{\"id\":\"0eef8214-4833-4089-9dff-b4138a14f196\",\"commitTime\":1609975977832}";
 
-  private UpdatesConfiguration createConfig(boolean usesLegacyManifest) {
+  private UpdatesConfiguration createConfig() {
     HashMap<String, Object> configMap = new HashMap<>();
     configMap.put("updateUrl", Uri.parse("https://exp.host/@esamelson/native-component-list"));
-    configMap.put("usesLegacyManifest", usesLegacyManifest);
     return new UpdatesConfiguration().loadValuesFromMap(configMap);
   }
 
   @Test
   public void testGetManifest_Legacy() throws JSONException {
+    ManifestResponse response = mock(ManifestResponse.class);
+    when(response.header("expo-protocol-version")).thenReturn(null);
+
     Manifest actual = ManifestFactory.getManifest(
       new JSONObject(legacyManifestJson),
-      null,
-      createConfig(true)
+      response,
+      createConfig()
     );
     Assert.assertTrue(actual instanceof LegacyManifest);
   }
 
   @Test
   public void testGetManifest_New() throws JSONException {
+    ManifestResponse response = mock(ManifestResponse.class);
+    when(response.header("expo-protocol-version")).thenReturn("0");
+
     Manifest actual = ManifestFactory.getManifest(
       new JSONObject(newManifestJson),
-      null,
-      createConfig(false)
+      response,
+      createConfig()
     );
+
     Assert.assertTrue(actual instanceof NewManifest);
   }
 
-  @Test
-  public void testGetEmbeddedManifest_Legacy() throws JSONException {
-    Manifest actual = ManifestFactory.getEmbeddedManifest(
-      new JSONObject(legacyManifestJson),
-      createConfig(true)
-    );
-    Assert.assertTrue(actual instanceof LegacyManifest);
-  }
```

Comment:
this was not deleted, just moved below the new `testGetManifest_Error` that git thinks is replacing it.